client/src/containers/insert/Insert.test.js

```diff
@@ -0,0 +1,38 @@
+import Enzyme, { render } from "enzyme";
```

Comment:
I think you can remove the commented lines of code in this file since they aren't being used